IpmiFeaturePkg/Library/IpmiCommandLib/IpmiCommandLibNetFnChassis.c

```diff
@@ -76,23 +76,35 @@ IpmiChassisControl (
   return Status;
 }
 
+/**
+  This function sets power restore policy.
+
+  @param[in]  SetPowerRestireRequest    The set power restore policy control
+                                        command request.
+  @param[out] SetPowerRestireResponse   The response of power restore policy.
+
+  @retval EFI_SUCCESS            Command is sent successfully.
+  @retval EFI_NOT_AVAILABLE_YET  Transport interface is not ready yet.
+  @retval Other                  Failure.
+
+**/
 EFI_STATUS
 EFIAPI
 IpmiSetPowerRestorePolicy (
-  IN  IPMI_SET_POWER_RESTORE_POLICY_REQUEST   *ChassisControlRequest,
-  OUT IPMI_SET_POWER_RESTORE_POLICY_RESPONSE  *ChassisControlResponse
+  IN  IPMI_SET_POWER_RESTORE_POLICY_REQUEST   *SetPowerRestireRequest,
+  OUT IPMI_SET_POWER_RESTORE_POLICY_RESPONSE  *SetPowerRestireResponse
   )
 {
   EFI_STATUS  Status;
   UINT32      DataSize;
 
-  DataSize = sizeof (*ChassisControlResponse);
+  DataSize = sizeof (*SetPowerRestireResponse);
   Status   = IpmiSubmitCommand (
                IPMI_NETFN_CHASSIS,
                IPMI_CHASSIS_SET_POWER_RESTORE_POLICY,
-               (VOID *)ChassisControlRequest,
-               sizeof (*ChassisControlRequest),
-               (VOID *)ChassisControlResponse,
+               (VOID *)SetPowerRestireRequest,
+               sizeof (*SetPowerRestireRequest),
+               (VOID *)SetPowerRestireResponse,

```

Comment:
typos on params. "Restore"

Reply:
These misspellings are coming from upstream IpmiCommandLib

https://github.com/tianocore/edk2/blob/master/MdeModulePkg/Include/Library/IpmiCommandLib.h#L412

I think we can get them fixed there, but it's a longer process.